resources/recipes.py

```diff
@@ -120,10 +120,11 @@ def plank(thing: str):
     rm.crafting_shaped('crafting/wattle', ['X', 'X'], {'X': '#minecraft:logs'}, (6, 'tfc:wattle')).with_advancement('#minecraft:logs')
     rm.crafting_shapeless('crafting/daub', ('tfc:straw', 'tfc:straw', 'tfc:straw', 'minecraft:clay_ball', 'minecraft:clay_ball', 'minecraft:clay_ball', '#minecraft:dirt', '#minecraft:dirt', '#minecraft:dirt'), (6, 'tfc:daub'))
     rm.crafting_shaped('crafting/composter', ['X X', 'XYX', 'XYX'], {'X': '#tfc:lumber', 'Y': '#minecraft:dirt'}, 'tfc:composter').with_advancement('#tfc:lumber')
+    # todo: this allows the bloomery to be crafted from any combo of sheets, instead of all of the same type. do we need separate recipes for each bronze type?
+    rm.crafting_shaped('crafting/bloomery', ['XXX', 'X X', 'XXX'], {'X': '#forge:double_sheets/any_bronze'}, 'tfc:bloomery').with_advancement('#tfc:bronze_anvils')
```

Comment:
Two things:
- The advancement should be based on the ingredients in the recipe, aka it should use `#forge:double_sheets/any_bronze`.
- I think letting it be crafted via any bronze sheets is fine tbh, so no, we wouldn't need three recipes.